casbin/model/model.cpp

```diff
@@ -119,7 +119,7 @@ Model* Model::NewModelFromFile(const string& path) {
 }
 
 Model*  Model::NewModelFromString(const string& text) {
-	Model* m = new Model();
+	Model* m =new Model();
 	m->LoadModelFromText(text);

```

Comment:
Keep the same style to construct an object. Use

```C++
Model m;
m->LoadModelFromText(text;
```

Reply:
OK,I will use unique_ptr instead of raw pointer.

Reply:
If model is a unique_ptr<Model>
Should I use  bool func(Model* m) and func(model.get())  or
 bool func(uniqueptr<Model> m) and func(model)

Reply:
unique_ptr<Model> in Enforcer means you store a pointer in Enforcer object that points to a Model's instance, doesn't affect how you define the Model class.

Reply:
I add unique_ptr for model and adapter in enforcer,you can have a look.

Reply:
I sent you a private message on gitter, we can have some discuss there.